src/server/Dockerfile

```diff
@@ -1,17 +1,19 @@
-# base image
-FROM node
+# Base image. Alpine 6 times smaller than just node
+FROM node:alpine
 
-# static server 
+# Static server
 RUN npm install -g serve
```

Comment:
What do we use it for?

Reply:
for statis server. not used now I think.